libmscore/fermata.cpp

```diff
@@ -83,7 +83,7 @@ bool Fermata::readProperties(XmlReader& e)
             setPlay(e.readBool());
       else if (tag == "timeStretch")
             _timeStretch = e.readDouble();
-      else if (tag == "offset") {
+      else if (tag == "offset" || tag == "minDistance") {
             if (score()->mscVersion() > 114)
                   Element::readProperties(e);
```

Comment:
I wrote this because I don't know the mscVersion of 3.1 (the first release min. distance got introduced)...

Reply:
If I am correct, handling `minDistance` explicitly here shouldn't be needed, `Element::readProperties` should probably handle it properly.

Reply:
Correct. I don't quite understand what does line 88 here mean, the "false" return doesn't get assigned to any variable?